src/app/pages/products/products.component.ts

```diff
@@ -1,27 +1,28 @@
 import { Component, OnInit } from '@angular/core';
-import { ProductInterface } from '../interfaces/product-interface';
+import { ProductInterface } from '@shared/interfaces/product-interface';
 import { ActivatedRoute } from '@angular/router';
-import { FiltersService } from '../services/filters.service';
-import { PaginationService } from '../services/pagination.service';
-import { RangeInterface } from '../interfaces/pagination-interfaces';
+import { FiltersService } from '@shared/services/filters.service';
+import { PaginationService } from '@shared/services/pagination.service';
+import { RangeInterface } from '@shared/interfaces/pagination-interfaces';
+import { SortingService } from '@shared/services/sorting.service';
 
 @Component({
-    selector: 'app-main-view',
-    templateUrl: './main-view.component.html',
-    styleUrls: ['./main-view.component.scss']
+    selector: 'app-products',
+    templateUrl: './products.component.html',
+    styleUrls: ['./products.component.scss']
 })
-export class MainViewComponent implements OnInit {
-
-    constructor(private route: ActivatedRoute, private filtersService: FiltersService, private paginationService: PaginationService) {
-    }
-
+export class ProductsComponent implements OnInit {
     public products: ProductInterface[] = [];
     public productsForPage: ProductInterface[] = [];
     private range: RangeInterface = {
         start: 0,
         end: 9
     };
 
+    constructor(private route: ActivatedRoute, private filtersService: FiltersService,
+                private paginationService: PaginationService, private sortingService: SortingService) {
```

Comment:
Bad indentation. I would suggest to hold each constructor inject per one line. It will help you if you get conflicts in constructor. Also, private staff better to name with `_` prefix.

```ts
constructor(
  private _route: ActivatedRoute,
  private _filtersService: FiltersService
}